src/main/scala/subsystem/HasTiles.scala

```diff
@@ -103,7 +103,7 @@ trait HasTiles { this: BaseSubsystem =>
     //    From CLINT: "msip" and "mtip"
     tile.crossIntIn() :=
       clintOpt.map { _.intnode }
-        .getOrElse { NullIntSource(sources = CLINTConsts.ints) }
+        .getOrElse { NullIntSource() }
```

Comment:
Not clear why this change is needed - no CLINT option works even without this change.